README.md

```diff
@@ -201,6 +216,19 @@ PS Code\vcpkg>         .\vcpkg install pthreads opencv[ffmpeg] #replace with ope
 
 9.  Open Powershell, go to the `darknet` folder and build with the command `.\build.ps1`. If you want to use Visual Studio, you will find two custom solutions created for you by CMake after the build, one in `build_win_debug` and the other in `build_win_release`, containing all the appropriate config flags for your system.
 
+### How to compile on Windows (using `Cmake-GUI`)
+
```

Comment:
maybe it’s worth adding:

Use Cmake-GUI if there are installed the most of requirements: [Darknet requirements](https://github.com/AlexeyAB/darknet#requirements)

Reply:
Sounds good, I added the blurb in the beginning of the vcpkg install method to CMake-GUI as well.